service/src/main/java/tds/config/repositories/impl/mappers/ClientSystemFlagRowMapper.java

```diff
@@ -0,0 +1,35 @@
+package tds.config.repositories.impl.mappers;
+
+import org.springframework.jdbc.core.RowMapper;
+import tds.config.ClientSystemFlag;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.time.ZoneOffset;
+
+/**
+ * Map a {@link ClientSystemFlag} from the database to a POJO.
+ */
+public class ClientSystemFlagRowMapper implements RowMapper<ClientSystemFlag> {
+    @Override
+    public ClientSystemFlag mapRow(ResultSet rs, int i) throws SQLException {
+        Instant dateChanged = rs.getTimestamp("dateChanged") == null
+                ? null
+                : rs.getTimestamp("dateChanged").toLocalDateTime().toInstant(ZoneOffset.UTC);
```

Comment:
Were we going to make a helper method in common at some point?  Maybe that will come later just calling it out.